BlazarUI/app/scripts/data/BranchApi.js

```diff
@@ -77,9 +78,9 @@ function triggerBuild(params, moduleIds, downstreamModules, resetCache, cb) {
   if (moduleIds === null) {
     moduleIds = [];
   }
-
+  const username = Cookies.get(config['usernameCookie']) || null;
   const buildPromise = new Resource({
-    url: `${config.apiRoot}/branches/builds/branch/${params.branchId}`,
+    url: `${config.apiRoot}/branches/builds/branch/${params.branchId}?username=${username}`,
```

Comment:
Is the behavior for Jackson that if you send a query param of the type `Optional` with a value of `null`, it's bound as `Optional.absent()`?